packages/type-utils/package.json

```diff
@@ -52,7 +52,11 @@
   },
   "devDependencies": {
     "@typescript-eslint/parser": "6.2.0",
-    "ajv": "^8.12.0",
```

Comment:
I had to downlevel this to match the rest of the workspace, otherwise `patch-package` hard errors on it